### DISCUSSION TITLE
May 22/2023 Update

### DISCUSSION THREAD

**Discussion by corygalyna** [ZORALab/Hestia#107] [Q&A] [answered]
@hollowaykeanho  update please.

**Comment by hollowaykeanho** [comment, marked as answer]
1. hestiaHUGO touch up with defensive checking. No more new development except bug busting during documentations porting.
2. ported `/en/` & `/zh-hans/` pages.
3. ported `/en/404/` & `/zh-hans/404/` pages.
4. currently porting `/en/licenses/` and `/zh-hans/licenses/` pages.

**Comment by corygalyna** [comment]
Nice... we should be on track by then.. any technical difficulty?

**Comment by hollowaykeanho** [comment]
No. Only page re-designs and translations. That's all. Estimating next week can release 1.2.0.

**Comment by corygalyna** [comment]
ok. Closing this sync up.
